settings.py

```diff
@@ -1,23 +1,71 @@
 """Settings used in rsync_to_rdisc.py"""
-wkdir = "/hpc/diaggen/data/upload/"
+
+""" General settings """
+wkdir = "/diaggen/data/upload/"
+temperror = "{}temp.error".format(wkdir)
 log = "Rsync_Dx.log"
 errorlog = "Rsync_Dx.errorlog"
-temperror = "/hpc/diaggen/data/upload/temp.error"
-bgarray = "/data/diaggen/DIT-bgarray" 
+bgarray = "/mnt/bgarray/"
 
-folder_dic = {
-    "Exomes":"/data/diaggen/DIT-bgarray/Illumina/Exomes", 
-    "TRANSFER":"/data/diaggen/DIT-bgarray/TRANSFER", 
-    "MIPS":"/data/diaggen/DIT-bgarray/Illumina/MIPS", 
-    "RAW_data_MIPS":"/data/diaggen/DIT-bgarray/RAW_data/MIPS", 
-    "RAW_data":"/data/diaggen/DIT-bgarray/RAW_data/BACKUP_TEMP", 
-    "Genomes":"/data/diaggen/DIT-bgarray/Illumina/Genomes"
-    }
 
-"""Mail finished transfer"""
-email_from = '<email>'
+""" Server/user settings """
+host_keys = ''
+server = ''
+user = ''
 
-finished_mail = '<email>'
 
-owner_dic = {'<username1>':'<email1>','<username2>':'<email2>'}
+"""Mail finished transfer"""
+email_from = ''
+email_to = ['', '']
+
 
+""" Transfer folders  """
```

Comment:
I like the new configuration dict. But would prefer to have te settings as a nested dic.
```python
folder_dic = {
     "Exomes": {
         "input": "/hpc/diaggen/data/upload/Exomes/",
         "output": "{}Illumina/Exomes/".format(bgarray),
         "files_required": "workflow.done",
         "continue_without_email": "False"
     }
}
```
Maybe a nice addition: Being to able to specify 'input' and 'output' location + server. This way we could also use the tool + server to perform transfers from/to other servers. Maybe a bit too much for this first PR though.

Reply:
Agree with dict. 
Other location/server is beyond scope for now.